### ISSUE TITLE
Avatar and Chip - Ensure content is visible to assistive technologies - (2036569782)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5563]
### Module: 
 07b Avatar and Chip 

### Violation: 
 Ensure content is visible to assistive technologies 

![image](https://user-images.githubusercontent.com/5023024/202801089-657d9090-0047-4bbe-8294-b61fc7c73c17.png)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036569782 

### Severity: 
 10 

### Description: 
 [Issue]
There are elements that are hidden for assistive technology users using aria-hidden arrtibute. Example included:
- mj below Test1. Medium Avatar
- mj below Test 3. Small Avatar

[User Impact]
Assistive technology users might miss the information which is provided.

[Code Reference] 
 ```
<span aria-hidden="true" class="initials">MJ</span>

 ``` 

### Note: 
 [Suggestion]
Developers should ensure that content is available for all the assistive technology users. Remove aria-hidden attribute from the elements.
Provide hidden text as "Michael Jordan" to make it descriptive.

[Complaint Code Example]
``` <span class="initials"> ```MJ
``` <span class="sr-only"> ```Michael Jordan``` </span> ```
``` </span> ```
 

### Media Type: 
 Page Structure 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Maybe we can expose text similar to the suggestion from a prop, possibly making it required?

**Comment by geospatialem** [comment]
Could be coupled with #5564

**Comment by geospatialem** [comment]
Completed as part of #5564.
